neuro_flow/expr.py

```diff
@@ -313,6 +314,37 @@ def make_text(arg: Token) -> Text:
     return Text(arg.start, arg.end, arg.value)
 
 
+@dataclasses.dataclass(frozen=True)
+class BinOp(Item):
+    op: Callable[[TypeT, TypeT], TypeT]
+    args: Tuple[Item, Item]
```

Comment:
Please use explicit *left* and *right* operands instead of 2-items tuple.
This is `BinOp` class, the number of arguments is always 2.